Formula/weechat.rb

```diff
@@ -48,5 +47,7 @@ def install
 
   test do
     system "#{bin}/weechat", "-r", "/quit"
+    system "#{bin}/weechat", "-r", "/script install multiline.pl; /quit"
```

Comment:
Change this line to:
```bash
system "#{bin}/weechat", "-r", "/script install multiline.pl; /wait 1 /quit"
```

Reply:
You could also use `xterm_paste.pl` instead of `multiline.pl`, since the former is 1/4th the size.

Reply:
@ericboehs Thanks, trying that!